installer/datafiles/ruby.data

```diff
@@ -3,38 +3,14 @@ RUBY_DEST:                '/opt/microsoft/omsagent/ruby'
 
 %Files
```

Comment:
There are a lot of harmless changes here. Some of these might be caused by sorting files in genRubyInstaller.sh. I won't review this, but will review genRubyInstaller.sh.

Did you diff this resulting file with the file I got by running genRubyInstaller on a non-universal system? You should do that, and I saved the file in case you want it (rather than regenerating yourself).

Given that I can't really review ruby.data, we need to be sure that there are no unintended changes here.